components/board.loading/R/loading_module_received.R

```diff
@@ -17,38 +17,81 @@ upload_module_received_server <- function(id,
     id, function(input, output, session) {
       ns <- session$ns ## NAMESPACE
 
-      refresh_table <- reactiveVal(0)
+      nr_ds_received <- reactiveVal(0)
 
-      ## ------------ get received files
-      getReceivedFiles <- shiny::reactive({
-        req(auth$logged)
-        if (!auth$logged) {
-          return(c())
+      # function that listed to input new_dataset_received
+      show_shared_tab <- function(){
+        if (input$new_dataset_received) {
+          bigdash.selectTab(session, "load-tab")
+          shinyjs::runjs('$("[data-value=\'Sharing\']").click();')
```

Comment:
@ESCRI11 I had a few crashes with this line before, since there may be situations where UI was rendered when the js line is run, and the app crashed. I tried several solutions (on document ready, only run if Sharing exists) but no good solution.

Since we have a message in yellow telling the user that we have shared datasets, I opted to not redirect to Sharing!

I could not reproduce (welcome page, load example data > introduce shared dataset > click to go to shared tab), and it worked.. not sure why its working now, but if it does not fail again, lets merge!

T